tests/index.test.ts

```diff
@@ -0,0 +1,32 @@
+import {assert} from "chai";
+import net from "net";
+import HobbitsP2PNetwork, {HobbitsOpts} from "../src";
+
+describe("P2P", () => {
+  const ctx: HobbitsOpts = {
+    networkId: 1,
+    chainId: 1,
+    state: {
+      latestFinalizedRoot: Buffer.alloc(0),
+      latestFinalizedEpoch: 1,
+      latestFinalizedSlot: 1
+    }
+  };
+  const server1 = new HobbitsP2PNetwork(ctx);
+  const server2 = new HobbitsP2PNetwork({...ctx, port: 9001, bootnodes: ["127.0.0.1:9000"]});
```

Comment:
Planning on fleshing out tests in this iteration or in a future one?

Reply:
I'll spend sometime this weekend adding to them! Get base cases for the majority of the functions